scripts/update/qbittorrent.sh

```diff
@@ -19,4 +19,9 @@ if [[ -f /install/.qbittorrent.lock ]]; then
         echo_info "qBittorrent systemd services have been updated. Please restart qBittorrent services at your convenience."
     fi
     #End systemd service updates
+    if ! grep -q "proxy_connect_timeout" /etc/nginx/apps/qbittorrent.conf; then
```

Comment:
is `proxy_connect_timeout` still a relevant test for if this needs to be set?

Reply:
This might make more sense @brettpetch?
```suggestion
    if ! grep -q "client_max_body_size" /etc/nginx/apps/qbittorrent.conf; then
```